sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/DefaultSchedulerTest.java

```diff
@@ -789,9 +796,9 @@ private void register() {
                 .setHostname(TestConstants.HOSTNAME)
                 .addAllResources(
                         Arrays.asList(
-                                ResourceTestUtils.getUnreservedCpu(TASK_A_CPU),
-                                ResourceTestUtils.getUnreservedMem(TASK_A_MEM),
-                                ResourceTestUtils.getUnreservedDisk(TASK_A_DISK)))
+                                ResourceTestUtils.getUnreservedCpu(TASK_A_CPU + 0.1),
```

Comment:
Ideally these 0.1/32/256 values should be consts somewhere. Looks like they're used here and in OfferTestUtils, maybe they could be placed in OfferTestUtils? Or are they in the main src somewhere as well?